Makefile

```diff
@@ -45,6 +50,44 @@ validate-infra-secrets: read-keyvault-config install-fetch-config set-azure-acco
 	bin/fetch_config.rb -s azure-key-vault-secret:${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} -d quiet \
 		&& echo Data in ${KEY_VAULT_NAME}/${KEY_VAULT_SECRET_NAME} looks valid
 
+.PHONY: set-space-developer
+set-space-developer: read-deployment-config ## `make dev set-space-developer USER_ID=first.last@digital.education.gov.uk`
+	$(if $(USER_ID), , $(error Missing environment variable "USER_ID", USER_ID required for this command to run))
+	cf set-space-role $(USER_ID) dfe $(SPACE) SpaceDeveloper
+
+.PHONY: unset-space-developer
+unset-space-developer: read-deployment-config ## `make dev unset-space-developer USER_ID=first.last@digital.education.gov.uk`
+	$(if $(USER_ID), , $(error Missing environment variable "USER_ID", USER_ID required for this command to run))
+	cf unset-space-role $(USER_ID) dfe $(SPACE) SpaceDeveloper
+
+stop-app: read-deployment-config ## Stops api app, `make dev stop-app CONFIRM_STOP=1`
+	$(if $(CONFIRM_STOP), , $(error stop-app can only run with CONFIRM_STOP))
+	cf target -s ${SPACE}
+	cf stop ${API_APP_NAME}
+
+get-postgres-instance-guid: read-deployment-config ## Gets the postgres service instance's guid
+	cf target -s ${SPACE} > /dev/null
+	cf service ${POSTGRES_DATABASE_NAME} --guid
+	$(eval DB_INSTANCE_GUID=$(shell cf service ${POSTGRES_DATABASE_NAME} --guid))
+
+rename-postgres-service: read-deployment-config ## `make dev rename-postgres-service NEW_NAME_SUFFIX=old CONFIRM_RENAME`
+	$(if $(CONFIRM_RENAME), , $(error can only run with CONFIRM_RENAME))
+	$(if $(NEW_NAME_SUFFIX), , $(error NEW_NAME_SUFFIX is required))
+	cf target -s ${SPACE} > /dev/null
+	cf rename-service  ${POSTGRES_DATABASE_NAME} ${POSTGRES_DATABASE_NAME}-$(NEW_NAME_SUFFIX)
+
+remove-postgres-tf-state: terraform-init ## `make dev remove-postgres-tf-state PASSCODE=XXX`
+	cd terraform && terraform state rm cloudfoundry_service_instance.postgres
+
+restore-postgres: terraform-init read-deployment-config ## `make dev restore-postgres DB_INSTANCE_GUID="<cf service db-name --guid>" BEFORE_TIME="" TF_VAR_api_docker_image=ghcr.io/dfe-digital/qualified-teachers-api:<COMMIT_SHA> PASSCODE=<auth code from https://login.london.cloud.service.gov.uk/passcode>`
```

Comment:
Can the time format be included, eg `BEFORE_TIME="yyyy-MM-dd hh:mm:ss"` (or whatever the correct format is)